packages/devops/scripts/utility/detectStage.sh

```diff
@@ -3,6 +3,6 @@
 # Get the stage tag of this ec2 instance from AWS
 TAG_NAME="Stage"
 INSTANCE_ID="$(wget -qO- http://instance-data/latest/meta-data/instance-id)"
-REGION="$(wget -qO- http://instance-data/latest/meta-data/placement/availability-zone | sed -e 's:\([0-9][0-9]*\)[a-z]*\$:\\1:')"
+REGION="`wget -qO- http://instance-data/latest/meta-data/placement/availability-zone | sed -e 's:\([0-9][0-9]*\)[a-z]*\$:\\1:'`"
```

Comment:
I don't know why this doesn't work... It was automatically applied by [shell-format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format&ssr=false#overview) 😢

Reply:
I'm not familiar with the `\` syntax for `sed`, I use `/`. In the meantime, I'll have a look at the PR which introduced this change to verify that every other related change is ok